diff --git a/arch/arm/mach-s3c64xx/dev-uart.c b/arch/arm/mach-s3c64xx/dev-uart.c
index f797f74..284f23e 100644
--- a/arch/arm/mach-s3c64xx/dev-uart.c
+++ b/arch/arm/mach-s3c64xx/dev-uart.c
@@ -102,29 +102,29 @@ static struct resource s3c6xx_uart2_resource[] = {
 	},
 };
 
-static struct resource s3c64xx_uart3_resource[] = {
-	[0] = {
-		.start = S3C_PA_UART3,
-		.end   = S3C_PA_UART3 + 0x100,
-		.flags = IORESOURCE_MEM,
-	},
-	[1] = {
-		.start	= IRQ_S3CUART_RX3,
-		.end	= IRQ_S3CUART_RX3,
-		.flags	= IORESOURCE_IRQ,
-	},
-	[2] = {
-		.start	= IRQ_S3CUART_TX3,
-		.end	= IRQ_S3CUART_TX3,
-		.flags	= IORESOURCE_IRQ,
-
-	},
-	[3] = {
-		.start	= IRQ_S3CUART_ERR3,
-		.end	= IRQ_S3CUART_ERR3,
-		.flags	= IORESOURCE_IRQ,
-	},
-};
+//static struct resource s3c64xx_uart3_resource[] = {
+//	[0] = {
+//		.start = S3C_PA_UART3,
+//		.end   = S3C_PA_UART3 + 0x100,
+//		.flags = IORESOURCE_MEM,
+//	},
+//	[1] = {
+//		.start	= IRQ_S3CUART_RX3,
+//		.end	= IRQ_S3CUART_RX3,
+//		.flags	= IORESOURCE_IRQ,
+//	},
+//	[2] = {
+//		.start	= IRQ_S3CUART_TX3,
+//		.end	= IRQ_S3CUART_TX3,
+//		.flags	= IORESOURCE_IRQ,
+//
+//	},
+//	[3] = {
+//		.start	= IRQ_S3CUART_ERR3,
+//		.end	= IRQ_S3CUART_ERR3,
+//		.flags	= IORESOURCE_IRQ,
+//	},
+//};
 
 
 struct s3c24xx_uart_resources s3c64xx_uart_resources[] __initdata = {
@@ -140,8 +140,8 @@ struct s3c24xx_uart_resources s3c64xx_uart_resources[] __initdata = {
 		.resources	= s3c6xx_uart2_resource,
 		.nr_resources	= ARRAY_SIZE(s3c6xx_uart2_resource),
 	},
-	[3] = {
-		.resources	= s3c64xx_uart3_resource,
-		.nr_resources	= ARRAY_SIZE(s3c64xx_uart3_resource),
-	},
+//	[3] = {
+//		.resources	= s3c64xx_uart3_resource,
+//		.nr_resources	= ARRAY_SIZE(s3c64xx_uart3_resource),
+//	},
 };
diff --git a/arch/arm/mach-s3c64xx/irq.c b/arch/arm/mach-s3c64xx/irq.c
index 67a145d..135e0bc 100644
--- a/arch/arm/mach-s3c64xx/irq.c
+++ b/arch/arm/mach-s3c64xx/irq.c
@@ -41,11 +41,11 @@ static struct s3c_uart_irq uart_irqs[] = {
 		.base_irq	= IRQ_S3CUART_BASE2,
 		.parent_irq	= IRQ_UART2,
 	},
-	[3] = {
-		.regs		= S3C_VA_UART3,
-		.base_irq	= IRQ_S3CUART_BASE3,
-		.parent_irq	= IRQ_UART3,
-	},
+//	[3] = {
+//		.regs		= S3C_VA_UART3,
+//		.base_irq	= IRQ_S3CUART_BASE3,
+//		.parent_irq	= IRQ_UART3,
+//	},
 };
 
 
diff --git a/arch/arm/mach-s3c64xx/mach-mini6410.c b/arch/arm/mach-s3c64xx/mach-mini6410.c
index afb5520..8992f03 100644
--- a/arch/arm/mach-s3c64xx/mach-mini6410.c
+++ b/arch/arm/mach-s3c64xx/mach-mini6410.c
@@ -104,13 +104,13 @@ static struct s3c2410_uartcfg mini6410_uartcfgs[] __initdata = {
 		.ulcon	     = ULCON,
 		.ufcon	     = UFCON,
 	},
-	[3] = {
-		.hwport	     = 3,
-		.flags	     = 0,
-		.ucon	     = UCON,
-		.ulcon	     = ULCON,
-		.ufcon	     = UFCON,
-	},
+//	[3] = {
+//		.hwport	     = 3,
+//		.flags	     = 0,
+//		.ucon	     = UCON,
+//		.ulcon	     = ULCON,
+//		.ufcon	     = UFCON,
+//	},
 };
 
 /* framebuffer and LCD setup. */
diff --git a/arch/arm/mach-s3c64xx/sleep.S b/arch/arm/mach-s3c64xx/sleep.S
index b2ef443..ec0c572 100644
--- a/arch/arm/mach-s3c64xx/sleep.S
+++ b/arch/arm/mach-s3c64xx/sleep.S
@@ -130,7 +130,7 @@ ENTRY(s3c_cpu_resume)
 	mcr	p15, 0, r6, c2, c0, 0	@ Translation Table BASE0
 	mcr	p15, 0, r7, c2, c0, 1	@ Translation Table BASE1
 	mcr	p15, 0, r8, c2, c0, 2	@ Translation Table Control
-	mcr	p15, 0, r10, c1, c0, 1	@ Auxiliary control register
+@	mcr	p15, 0, r10, c1, c0, 1	@ Auxiliary control register
 
 	mov	r0, #0			@ restore copro access controls
 	mcr	p15, 0, r11, c1, c0, 2	@ Co-processor access controls
